### PR TITLE
Demo App Support for Injection of Order Id and Access Token

### PR DESCRIPTION
**Pull request by KunJeongPark**
### Summary of changes

- Added InjectedValues enum in `DemoMerchantAPI`
- Added checking for non-nil values in InjectedValues in getOrderID and getAccessToken functions.

### Checklist

~- [ ] Added a changelog entry~

### Authors
@KunJeongPark @sshropshire

### CODE REVIEW

**Review by jcnoriega — APPROVED** [review]

File: Demo/Demo/Networking/DemoMerchantAPI.swift [hunk 1]
```diff
@@ -9,6 +9,12 @@ final class DemoMerchantAPI {
 
     static let sharedService = DemoMerchantAPI()
     var accessToken: String?
+    
+    enum InjectedValues {
+        static let orderID: String? = nil
+        static let accessToken: String? = nil
+    }
+
```
Comment (scannillo) [inline Demo/Demo/Networking/DemoMerchantAPI.swift:18]:
```suggestion
```

File: Demo/Demo/Networking/DemoMerchantAPI.swift [hunk 2]
```diff
@@ -9,6 +9,12 @@ final class DemoMerchantAPI {
 
     static let sharedService = DemoMerchantAPI()
     var accessToken: String?
+    
+    enum InjectedValues {
```
Comment (scannillo) [inline Demo/Demo/Networking/DemoMerchantAPI.swift:14]: Same comment as I left on Android [here](https://github.com/paypal/Android-SDK/pull/142#discussion_r1198030030) regarding adding some comments around making this more clear for less experienced devs to use.

File: Demo/Demo/Networking/DemoMerchantAPI.swift (hunk 1 above)
Comment (KunJeongPark) [reply Demo/Demo/Networking/DemoMerchantAPI.swift:18]: fixed here: adee603805692b65d9586ddc143a8b5e41b2a69b

File: Demo/Demo/Networking/DemoMerchantAPI.swift (hunk 2 above)
Comment (KunJeongPark) [reply Demo/Demo/Networking/DemoMerchantAPI.swift:14]: Added comment here adee603805692b65d9586ddc143a8b5e41b2a69b

**Review by scannillo — APPROVED** [review]

**Pull request merged**